service/worker/migration/workflow.go

```diff
@@ -62,10 +64,12 @@ const (
 type (
 	ForceReplicationParams struct {
 		Namespace               string
-		SkipAfterTime           time.Time // skip workflows that are updated after this time
-		ConcurrentActivityCount int32
-		RemoteCluster           string // remote cluster name
+		Query                   string // query to list workflows for force replicaiton
```

Comment:
replication

what makes it "force replication" as opposed to just replication?